### ISSUE TITLE
Problem

### ISSUE THREAD

**Issue by rppf** [Stolz/laravel-schema-spy#1]
I can't access jarfile schemaSpy.jar

Unable to access jarfile schemaSpy.jar

**Comment by Stolz** [comment]
Please provide more information. Which version of the connector are you using? Where did you installed it? How did you configured the **spy.php** file?

**Comment by rppf** [comment]
Thanks but I already fix that.

I have a new problem.

When I execute the command php artisan db:spy. An error will occur. The error is:
java.lang.IndexOutOfBoundsException: Index: 2, Size: 2

**Comment by Stolz** [comment]
That seems to be a problem with your JVM. Sorry but I'm afraid I can't help you as is not PHP related

**Comment by NicolasReibnitz** [comment]
I had the same problem. But echoing out the command that was generated in Command.php I could see what went wrong: I had no password set for my local database. I had to comment out this line:
`$parameters['-p']    = $connections[$connection]['password'];`
also in Command.php (line 52). That did the trick for me.

As a side note, I had to use 'mysql-connector-java-5.1.6'. The one I got from your link ('mysql-connector-java-8.0.15') did not work.

I hope this helps.

Cheers!
